ui/src/app/base/components/node/HardwareCard/HardwareCard.tsx

```diff
@@ -0,0 +1,72 @@
+import { Card } from "@canonical/react-components";
+
+import LabelledList from "app/base/components/LabelledList";
+import type { ControllerDetails } from "app/store/controller/types";
+import type { MachineDetails } from "app/store/machine/types";
+
+type Props = {
+  node: ControllerDetails | MachineDetails;
+};
+
+const HardwareCard = ({ node }: Props): JSX.Element => {
+  return (
+    <Card>
+      <strong className="p-muted-heading u-sv1">Hardware information</strong>
+      <hr />
+      <span className="u-sv1">System</span>
```

Comment:
This file is mostly unchanged, apart from the new name of the component (`SystemCard` to `HardwareCard`), the props API (`systemId` to `node`) and this highlighted section.